src/rdo.rs

```diff
@@ -1011,7 +1011,7 @@ pub fn rdo_mode_decision<T: Pixel>(
 
             (
               luma_mode,
-              get_sad(
+              get_satd(
```

Comment:
This is going into a variable called `sads` which will need to be renamed to `satds`. Or some other, better variable name. Naming is hard.

Reply:
Fixed. Using the wrong name is a Theora tradition that I don't want to continue :)